app/views/v2/edit_liability_ruling.scala.html

```diff
@@ -75,10 +80,12 @@
 
   @confirm_or_cancel_buttons(
    id = "edit_liability_decision",
-   confirmText = messages("case.v2.liability.save_and_continue_button"),
-   cancelText = messages("case.v2.liability.cancel_button"),
+   confirmText = "Save changes",
```

Comment:
should we aim to keep all messages in the message files, especially for when it comes to translating them

Reply:
I added messages for the confirm text and cancel text

Reply:
Thanks @oliwiaKot